radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/epoch/EpochManager.java

```diff
@@ -307,6 +298,8 @@ private void processLedgerUpdate(LedgerUpdate ledgerUpdate) {
 	private void processEpochChange(EpochChange epochChange) {
 		// Sanity check
 		if (epochChange.getEpoch() != this.currentEpoch() + 1) {
+			//TODO: looks like byzantine message, might be used for slashing?
+			//TODO: don't trow exception
```

Comment:
No not a byzantine message, `EpochChange` is a local message. This is a sanity check and we should keep the exception.

Reply:
done